### ISSUE TITLE
Cannot transfer between Windows 10 PC and iPhone XS iOS 14.5.1

### ISSUE THREAD

**Issue by HyperdrivenAU** [RobinLinus/snapdrop#300]
HI,

I use Snapdrop as my go-to file transfer program to move images from phone to my PC. However, since 14.5 or 14.5.1, it no longer works.

I get a connection between the devices, but when I try to send some images (or an image) nothing happens.

Messaging doesn't work either, in either direction.

Any ideas..??

Ian

**Comment by Bellisario** [comment]
No, for now you can find solutions only in these other problems: #258 #259 #252 #249 #247 #233 #261 #298
